content/02.intro.md

```diff
@@ -21,6 +22,8 @@ To make deep learning more accessible to biological researchers, we solicited in
 These individuals collaboratively contributed to this manuscript's writing using the GitHub version control platform [@url:https://github.com/Benjamin-Lee/deep-rules] and the Manubot manuscript generation toolset [@url:https://greenelab.github.io/meta-review/].
 The goal was to articulate a practical, accessible, and concise set of guidelines and suggestions for biologically oriented researchers to follow when using deep learning.
```

Comment:
```suggestion
The goal was to articulate a practical, accessible, and concise set of guidelines and suggestions for biologically oriented researchers to follow when using deep learning (Figure @fig:overview-fig).
```

This figure placement looks good to me, but I think we need to reference it in the text as well.

Reply:
Good shout, done.